<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableInput</class>
 <widget class="QWidget" name="TableInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableUp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>681</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableDown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>681</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClose">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>560</y>
     <width>77</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DEscription</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMain">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Main Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
